### DISCUSSION TITLE
sftp test container with privateKey auth

### DISCUSSION THREAD

**Discussion by WrettySV** [testcontainers/testcontainers-java#6431] [Q&A]
Help please. 
How to initialize container with private and public keys for specific user?
It does not work.  ERROR: com.jcraft.jsch.JSchException: Auth fail 
    protected static final GenericContainer sftpContainer = new GenericContainer<>(DockerImageName.parse("atmoz/sftp:latest"))
            .withExposedPorts(22)
            .withCopyFileToContainer(MountableFile.forHostPath("src/test/resources/key/testKey"),
                    "/home/tester/.ssh/ed/id_rsa")
            .withCopyFileToContainer(MountableFile.forHostPath("src/test/resources/key/testKey.pub"),
                    "/home/tester/.ssh/ed/id_rsa.pub")
            .withCommand("tester::1001");

**Comment by eddumelendez** [comment]
Duplicated https://github.com/testcontainers/testcontainers-java/discussions/6418
